apps/forum/vars/vault/main.yml.j2

```diff
@@ -0,0 +1,15 @@
+# secret: forum
+# customer: {{ customer }}
+# env_type: {{ env_type }}
+
+# MongoDB
+{% set mongodb_credentials = mongodb | json_query("[?release=='" ~ forum_mongodb_version ~ "'].databases | [0][?application=='forum'].{user: user, password: password, name: name} | [0]") %}
+MONGODB_ADMIN_PASSWORD: password
+MONGODB_USER: {{ mongodb_credentials.user }}
+MONGODB_PASSWORD: {{ mongodb_credentials. password }}
+MONGODB_DATABASE: {{ mongodb_credentials.name }}
+
+# Forum
+# Nota bene: the forum API_KEY should match the following edxapp lms settings:
+# COMMENTS_SERVICE_KEY
```

Comment:
Add a "FIXME" comment here.

Reply:
I don't know what I can say more than the existing comment.